README.md

```diff
@@ -68,6 +73,7 @@ new ProfileNetworthCalculator(profileData: object, museumData?: object, bankBala
 ```typescript
 getNetworth(options?: NetworthOptions): Promise<NetworthResult>
 getNonCosmeticNetworth(options?: NetworthOptions): Promise<NetworthResult>
+fromPreParsed(profileData: object, items: Items, bankBalance: number): ProfileNetworthCalculator;
```

Comment:
Almost feels like this should be under constructor, but it it a static method so i would move to its own. Idk